models/UserManager.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+class UserManager extends Model
+{
+    // Fonction permettant de récupérer le nom d'un admin pour un article précis.
+    function article_admin(){
+     $req = $this->db->query("
+            SELECT  
+                    admins.name
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [article_admin accesses the super-global variable $_GET.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)